src/Avatar.js

```diff
@@ -1,14 +1,33 @@
-import React from 'react';
-
-import './App.css';
-
-export function Avatar({avatar, onErrorLoad}) {
-    return (
-        <img 
-            class="avatar-image" 
-            src={avatar} 
-            alt=""
-            onError={onErrorLoad}
-        />
-    )
+import React, {useState} from 'react';
+import {PlaceholderAvatar} from './PlaceholderAvatar';
+
+export function Avatar({avatar, name}) {
+    const [isErrorLoadImg, setIsErrorLoadImg] = useState(false);
+
+    let initials = name && name.trim().split(' ', 2).map(n => n[0]).join('').toUpperCase();
+
+    function onErrorLoad() {
+        setIsErrorLoadImg(true);
+    }
```

Comment:
since you know jooks plese use `useCallback` here (don't forget about arrow function)